### PR TITLE
Update troubleshoot.md to use new dart tool

### PR DESCRIPTION
**Pull request by woinbo**
Update troubleshoot.md to use new dart tool related to issue #2617

@kwalrath I didn't asked you to work on this file I found it and did worked on this files so creating PR please don't mind for not taking permission from you 😅

### CODE REVIEW

**Review by Abhishek01039 — APPROVED** [review]
LGTM!

File: src/tools/pub/troubleshoot.md [hunk 1]
```diff
@@ -101,25 +101,25 @@ $ $Env:https_proxy="username:password@hostname:port"
...
    (`http://localhost:<port>?code=...`)
    but complains that the URL isn't reachable.
 3. Copy the _new localhost URL_  from the browser.
 4. In another terminal window in the same container or on the same host
-   as the one where `pub publish` was called, use the `curl` command to
+   as the one where `dart pub publish` was called, use the `curl` command to
    complete sign-in using the _new localhost URL_:
 
-   ```terminal
+```terminal
```
Comment (kwalrath) [inline src/tools/pub/troubleshoot.md:123]: I think this indentation was intentional, so that the command would be visible in step 4.

**Comment by kwalrath** [comment]
> Update troubleshoot.md to use new dart tool related to issue #2617
> 
> @kwalrath I didn't asked you to work on this file I found it and did worked on this files so creating PR please don't mind for not taking permission from you 😅

No problem at all! Thanks for fixing the issue. LGTM once you fix the indentation.

**Comment by woinbo** [comment]
> > Update troubleshoot.md to use new dart tool related to issue #2617
> > @kwalrath I didn't asked you to work on this file I found it and did worked on this files so creating PR please don't mind for not taking permission from you 😅
> 
> No problem at all! Thanks for fixing the issue. LGTM once you fix the indentation.

Done, Thank you!

**Comment by woinbo** [comment]
@kwalrath have a look at this PR too #2877

**Review by kwalrath — APPROVED** [review]

**Comment by kwalrath** [comment]
> @kwalrath have a look at this PR too #2877

I'm way behind on reviews (and work), due to other work I've needed to do... but I'll try to catch up!

**Pull request merged**